ci_handbook/modularity.qmd

```diff
@@ -0,0 +1,12 @@
+---
+title: "{{< fa arrow-right >}} Modularity"
+format:
+  html:
+    toc: true
+    toc-depth: 3
+    toc-position: left
+reading-time: true
+image: "../images/hive.jpg"
```

Comment:
Should be showing lego pieces
/images/Lego_dimensions.svg.png"

Reply:
keeping as is